src/Entity/Dossier.php

```diff
@@ -220,9 +220,9 @@ public function move(?Dossier $parentFolder): void
     {
         $this->setDossierParent($parentFolder);
 
-        if ($parentFolder && $parentFolder !== $this) {
+        if (true === $parentFolder?->getBPrive() && $parentFolder !== $this) {
             $parentFolder->addSousDossier($this);
-            if ($parentFolder->getBPrive() !== $this->getBPrive()) {
+            if (!$this->getBPrive()) {
                 $this->toggleVisibility();
             }
```

Comment:
I think there is a logic pb here, we should call `->addSousDossier()` even if `$parentFolder` is public